kannada/soochike.md

```diff
@@ -1,4 +1,5 @@
 ಇದು ಕನ್ನಡ ಬರಹಗಳ ಸೂಚಿಕೆ.
 
 ಲೇಖನಗಳು:
-1. ರೋಮಾಂಚನವೀ ಕನ್ನಡ! ಓದಲು, [ಇಲ್ಲಿ ಒತ್ತಿ](romaanchanavee_kannada/romaanchanavee_kannada.md).
\ No newline at end of file
+1. ರೋಮಾಂಚನವೀ ಕನ್ನಡ! ಓದಲು, [ಇಲ್ಲಿ ಒತ್ತಿ](romaanchanavee_kannada/romaanchanavee_kannada.md).
+2. ಕನ್ನಡ ಕಲಿಯಲು [ಇಲ್ಲಿ](kannada_kali/kannada_kali.md).
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.